source/core/slang-performance-profiler.cpp

```diff
@@ -39,7 +39,7 @@ namespace Slang
             for (auto func : data)
             {
                 out << func.key << ": \t";
-                out << func.value.invocationCount << "\t" << func.value.duration.count()/1000000 << "\n";
+                out << static_cast<Int32>(func.value.invocationCount) << "\t" << static_cast<double>(func.value.duration.count()/1000000) << "\n";
```

Comment:
Shouldn't this be more like
```
static_cast<double>(func.value.duration.count())/1000000
```
I think the division should happen after the type is changed to double; not before.

Reply:
You are right, I've completely misread the entire message, further testing with godbolt (https://godbolt.org/z/zYG75axMq), I will be pushing the latest commit.